Rmarkdown/Multiple_Ascending_Dose_PD_binary.Rmd

```diff
@@ -1,303 +1,271 @@
+
+```{r, cache = TRUE, fig.width = 10, fig.height = 4, warning = FALSE, message= FALSE} 
+
+  gg <- ggplot(data = pd_data, aes(x = NOMTIME, y = LIDV, color = TRTACT_high2low))
+  gg <- gg + xgx_stat_ci(distribution = "binomial")
+  gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                    units_plot    = time_units_plot)
+  gg <- gg + scale_y_continuous(labels=scales::percent)
+  gg <- gg + labs(y = pd_label, color = trtact_label)  
+  gg <- gg + xgx_annotate_status(status)
+  #gg <- gg + xgx_annotate_filenames(dirs)
+  #if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+  #xgx_save(width,height,dirs,"filename_main",status)
+  print(gg)
+
+```
+
+```{r, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message= FALSE}
+
+  gg <- ggplot(data = pd_data, aes(x=PROFDAY,y=LIDV))
+  gg <- gg + geom_point(aes(x=DAY_jitter,y=LIDV_jitter), size=2, alpha = 0.3)
+  gg <- gg + xgx_stat_ci(conf_level = 0.95, distribution = "binomial", geom = list("line", "ribbon"))
+
+  gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                    units_plot    = time_units_plot)
+  gg <- gg +  scale_x_continuous(breaks = seq(-1,8,1))
+  gg <- gg + scale_y_continuous(labels=scales::percent)
+  gg <- gg + labs(y = pd_label, color = trtact_label)  
+  gg <- gg + facet_grid(~TRTACT_low2high)
+  gg <- gg + xgx_annotate_status(status)
+  #gg <- gg + xgx_annotate_filenames(dirs)
+  #if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+  #xgx_save(width,height,dirs,"filename_main",status)
+  print(gg)
+```
+
+## Explore variability
+
+### Spaghetti plots of binary response over time, faceted by dose
+
+Use spaghetti plots to visualize the extent of variability between individuals. The wider the spread of the profiles, the higher the between subject variability. Distinguish different doses by color, or separate into different panels. 
+
+Binary spaghetti plots may not be as informative as spaghetti plots for continuous data, however you can pick out the transition states from 0 to 1 and back. By looking at the transition states, ask yourself these questions: For each dose group, at what time have most of the subjects "responded"? How wide is the spread of "response time"" across subjects (do all of them respond at the exact same time, or over a range of time?) Do you detect any dampening, or transitioning of response back to no response?
+
+```{r, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message= FALSE}
+gg <- ggplot(data = pd_data, 
+             aes(x=DAY_jitter,y=LIDV_jitter))
+gg <- gg + geom_point( size=2, alpha = 0.3)
+gg <- gg + geom_line(aes(group = ID), alpha = 0.3) 
+gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))

```

Comment:
don't need guides for color or fill in this plot

Reply:
removed now